### PR TITLE
Gatsby - Supergrid

### LINKED ISSUE #184: Migrate Supergrid page

### PR DESCRIPTION
**Pull request by jfidiles**
#### Description

Supergrid migration from Jekyll to Gatsby

#### Issues closing

Closes #184

### CODE REVIEW

**Comment by peduarte** [comment]
Deploy preview for *fathomlondon* ready!

Built with commit 76c2b78eff85053e41779e6d0e1372b5f7e0cf56

https://deploy-preview-205--fathomlondon.netlify.com

File: src/pages/supergrid.js [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+									<img
+										src={inpageGridA}
+										className="w-100 w-75-m"
+										alt="supergrid-inpage-grid-a"
+									/>
+								</figure>
+							</div>
+						</div>
+
+						{/* TODO: Replace this with wallop once it's created */}
```
Comment (peduarte) [inline src/pages/supergrid.js]: You should be able to use wallop already.

Take a look at @AndrewJDick's PR to see how he used it on the About page https://github.com/fathomlondon/fathomlondon.com/pull/203/files

**Comment by jfidiles** [comment]
Cool. Will add it! I haven’t added it because I remember you mentioned that we should do it in a more react way using state

**Comment by AndrewJDick** [comment]
Wallop implementation aside, this looks good 👍🏽

**Pull request merged**